sway/desktop/output.c

```diff
@@ -204,6 +204,36 @@ void output_layer_for_each_surface(struct sway_output *output,
 		output_surface_for_each_surface(output, wlr_layer_surface_v1->surface,
 			layer_surface->geo.x, layer_surface->geo.y, iterator,
 			user_data);
+
+		struct wlr_xdg_popup *state;
+		wl_list_for_each(state, &wlr_layer_surface_v1->popups, link) {
```

Comment:
We have `wlr_layer_surface_v1_for_each_surface` for this. But oh well, w/e

Reply:
I was going to use this but there were Problems and an hour in I gave up.